init.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
```

Comment:
Three variants to check:
- Without parameter --> Interactive mode
- Three parameters --> direct execution mode
- Else --> Show info how to use this script